<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daan Dekoning Krekels</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>214</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="GezichtenUitknippen">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecteer een map met afbeeldingen waarvan de gezichten moeten uitgeknipt worden. Selecteer een map waar de uitgeknipt gezichten opgelsagen worden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Gezichten Uitknippen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TrainenGezichten">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Train het systeem op gezichten die onderverdeeld zijn in mappen. Importeer of exporteer de gezichtsdata.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Trainen op Gezichten</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LijstMaken">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Genereer een aanezigheislijst aan de hand van een fofo of duid de personen in een foto aan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Aanwezigheidslijst Maken</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>483</width>
           <height>140</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStartGezichtenUitknippen">
   <property name="text">
    <string>StartGezichtenUitknippen</string>
   </property>
  </action>
  <action name="actionStartTrainen">
   <property name="text">
    <string>StartTrainen</string>
   </property>
  </action>
  <action name="actionStartLijstMaken">
   <property name="text">
    <string>StartLijstMaken</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>GezichtenUitknippen</sender>
   <signal>clicked()</signal>
   <receiver>actionStartGezichtenUitknippen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TrainenGezichten</sender>
   <signal>clicked()</signal>
   <receiver>actionStartTrainen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LijstMaken</sender>
   <signal>clicked()</signal>
   <receiver>actionStartLijstMaken</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
